media/css/firefox/home/master.scss

```diff
@@ -206,7 +206,8 @@ $brand-theme: 'firefox';
         }
     }
 
-    @media #{$mq-md} {
+    // note: this can be changed to md breakpoint once latest Protocol changes are merged
+    @media #{$mq-lg} {
```

Comment:
There was a legibility issue on medium screen sizes with the background image and white text.

![Screen Shot 2021-08-23 at 3 31 32 PM](https://user-images.githubusercontent.com/19650432/130467023-a3963dbe-01ff-4cb5-bafe-349a25bc5cd5.png)

Reply:
Good catch! I missed that completely.